### DISCUSSION TITLE
DreamBerd has been renamed to C

### DISCUSSION THREAD

**Discussion by TodePond** [TodePond/GulfOfMexico#284] [Announcements]
DreamBerd has been renamed to C.

![image](https://github.com/TodePond/C/assets/15892272/d0ff56f2-5e50-4be5-b94f-f0f38aa7df72)

Thanks for reading this week's release notes.

<hr /><em>This discussion was created from the release <a href='https://github.com/TodePond/C/releases/tag/v0.9c'>DreamBerd has been renamed to C</a>.</em>

**Comment by FlooferLand** [comment]
Very cool 👍
Hopefully no one forks this language and adds a lot of features nobody asked for

**Reply by lifr0m** [reply]
One dane gonna dislike this.

**Comment by silvasch** [comment]
I think this is a very bad decision, as DreamBerd had a lot of brand value in it's name. It has created a lot of new words, like to dreamberd, to be a dreamberdy person, and so on. I'm going to keep calling it DreamBerd.

**Reply by FlooferLand** [reply]
Have to agree on this one. This project is also where the term "dreaming" was coined from, which is a very popular word used by many, especially in the tech industry.

**Reply by jerbear2008** [reply]
I agree too. Now what will happen when we sleep at night, now that dreaming has been officially deprecated?

**Reply by taitep** [reply]
C:ing

**Comment by sharunkumar** [comment]
![image](https://github.com/TodePond/C/assets/715417/c4a1ac06-2272-4aa2-a91f-f937e8b1f76c)

**Comment by qexat** [comment]
I'm a bit confused by that, no explanations left?

**Comment by tonyaellie** [comment]
have you considered renaming to DreamCBird

**Reply by thegatesdev** [reply]
CreamBerd would also be a viable alternative.

**Comment by kitgore** [comment]
the most obvious next step

**Comment by Alexander-Popescu** [comment]
I think the major implications of this patch are going unnoticed to the general Dreamberd community, allow me to clarify.

Since its inception, the C programming language has carried the computer science industry on its shoulders. By associating itself with this key player in the beginning of the modern computing era, the Dreamberd foundation is conveying how C (formerly Dreamberd) is the only indefinite solution for the future computer programming languages. 

Still not convinced? Digest the provided mathematical proof:

C -> the C programming language
C -> C (formerly Dreamberd)

Now flip C

cↄ

Squint your eyes a little bit

∞

Thus, Dreamberd will last forever, as we all knew already.

**Comment by TSinChen** [comment]
I think "X" is good too, before "someone" take it

**Comment by cowtoolz** [comment]


**Reply by stohrendorf** [reply]
What about APL? "Another Programming Langunage"?

**Reply by Kirirou** [reply]
I think there is a company that sounds similar and has a logo that has been bitten.

**Reply by TodePond** [reply]
@rotteegher Yes you're right, it was briefly discussed on this podcast I was on: https://futureofcoding.org/episodes/064

**Reply by stohrendorf** [reply]
Then this company is a fraught, see https://en.wikipedia.org/wiki/APL_(programming_language)

**Reply by TodePond** [reply]
@stohrendorf Yes you're right, it was briefly discussed on this podcast I was on: https://futureofcoding.org/episodes/064

**Reply by stohrendorf** [reply]
@TodePond Yes I'm right, it was briefly discussed on this podcast you were on: https://futureofcoding.org/episodes/064

**Comment by skifli** [comment]
Well I didn't C that one coming...

**Comment by TheGB0077** [comment]
### I feel disappointed... To me, it's clear that only ℙ would be acceptable to represent the _𝕡𝕖𝕣𝕗𝕖𝕔𝕥_ language.

**Comment by Moosems** [comment]
..... Why?

**Reply by Kirirou** [reply]
You can't C ?

**Comment by notpushkin** [comment]
I c what you did there

**Comment by adri326** [comment]
You definitely have to trademark that logo or some billionaire might try to snatch it!
